index.html

```diff
@@ -2373,6 +2372,15 @@
         However, this information is also dependent on the user's local network
         context, so the risk is minimized.
       </p>
+      <p>
+        The API enables <a href=
+        "#monitoring-the-list-of-available-presentation-displays">monitoring
+        the list of available presentation displays</a>. How the user agent
+        determines the compatibility and availability of a presentation display
+        with a given URL is an implementation detail. This feature can be used
+        to probe information about DIAL applications the user has installed on
```

Comment:
Maybe, 

> If a <a>controlling user agent</a> can match a <a>presentation request URL</a> to a <a>DIAL</a> application and determine its availability, this feature can be used to probe information about which applications ...

Also, add a reference for DIAL?

http://www.dial-multiscreen.org/